package.json

```diff
@@ -31,17 +31,18 @@
   },
   "homepage": "https://github.com/codesect/stylelint-config#readme",
   "dependencies": {
+    "stylelint-a11y": "^1.2.3",
     "stylelint-config-standard": "^19.0.0",
-    "stylelint-scss": "^3.12.1"
+    "stylelint-scss": "^3.18.0"
   },
   "devDependencies": {
-    "eslint": "^6.6.0",
-    "eslint-config-prettier": "^6.7.0",
-    "jest": "^24.9.0",
-    "prettier": "^1.19.1",
-    "stylelint": "^12.0.0"
+    "eslint": "^7.12.1",
+    "eslint-config-prettier": "^6.15.0",
+    "jest": "^26.6.1",
+    "prettier": "^2.1.2",
+    "stylelint": "^13.7.2"
   },
   "peerDependencies": {
-    "stylelint": "^11.0.0 || ^12.0.0"
+    "stylelint": "^13.7.2"
```

Comment:
You should also include the previous versions

Reply:
Fixed :)